### PR TITLE
Fix Kasket double quoting array of IDs

### PR DESCRIPTION
**Pull request by pschambacher**
/cc @grosser @bquorning @kassio @craig-day

### CODE REVIEW

File: lib/kasket/visitor.rb [hunk 1]
```diff
@@ -144,6 +144,7 @@ def visit_FalseClass(_node)
     end
 
     def quoted(node)
+      return node if node.is_a?(String)
```
Comment (kassio) [inline lib/kasket/visitor.rb]: humm.. why?

Comment (kassio) [reply lib/kasket/visitor.rb]: forget it. half sleeping yet.

Comment (pschambacher) [reply lib/kasket/visitor.rb]: We end up generating invalid SQL where strings are quoted twice

**Comment by grosser** [comment]
❤️ nice fix :D

File: lib/kasket/visitor.rb (hunk 1 above)
Comment (bquorning) [reply lib/kasket/visitor.rb]: Hmm, it looks like quoting a string works identically in Rails 3 and 4:
```rb
# Loading development environment (Rails 3.2.22.4)
ActiveRecord::Base.connection.quote("foo")
#=> "'foo'"

# Loading development environment (Rails 4.2.10)
ActiveRecord::Base.connection.quote("foo")
#=> "'foo'"
```

**Pull request merged**